File: 백준/Silver/10431. 줄세우기/줄세우기.py
T = int(input())
for _ in range(T) :
    temp = list(map(int,input().split()))
    cnt = 0
    
    tc = temp[0]
    students = temp[1:]

    for i in range(20) :
        for j in range(i) :
            if students[j] > students[i] :
                cnt += 1

    print(tc, cnt)

File: 백준/Silver/10431. 줄세우기/줄세우기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st;

        int P = Integer.parseInt(br.readLine());
        for (int tc = 1; tc <= P ; tc++) {
            st = new StringTokenizer(br.readLine());
            int num = Integer.parseInt(st.nextToken());
            int[] students = new int[20];
            int cnt = 0;

            for (int idx = 0; idx < 20; idx++) {
                students[idx] = Integer.parseInt(st.nextToken());
            }

            for (int i = 0; i < 20; i++) {
                for (int j = 0; j<i; j++) {
                    if (students[j] > students[i]) {
                        cnt++;
                    }
                }
             }

            System.out.println(tc +" " + cnt);
        }

    }

}
